src/mjolnir/linkclassification.cc

```diff
@@ -227,7 +227,11 @@ std::pair<uint32_t, uint32_t> Reclassify(LinkTreeNode& root,
       sequence<Edge>::iterator element = edges[idx];
       auto edge = *element;
       if (rc > edge.attributes.importance) {
-        edge.attributes.importance = rc;
+        if (rc < static_cast<uint32_t>(RoadClass::kUnclassified))
+          edge.attributes.importance = rc;
+        else
+          edge.attributes.importance = static_cast<uint32_t>(RoadClass::kTertiary);
```

Comment:
whats this doing?

Reply:
comments were in the description.."Updated reclassify link logic to allow residential roads to be added to the tree, but we only downgrade the links to tertiary."